### PR TITLE
Changes Dockerfile to use go 1.4

### PR DESCRIPTION
**Pull request by mrunalp**
Ran the tests locally using the image based off golang:1.4 and they succeeded.

Signed-off-by: Mrunal Patel <mrunalp@gmail.com> (github: mrunalp)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM crosbymichael/golang
+FROM golang:1.4
 
 RUN apt-get update && apt-get install -y gcc make
```
Comment (tianon) [inline Dockerfile]: These two should be in the base image already now, aren't they? :+1:

Comment (tianon) [inline Dockerfile]: https://github.com/docker-library/golang/blob/184585232ab4d65ad4acbbd564ac648d7e6440b2/1.4/Dockerfile#L5

Comment (mrunalp) [inline Dockerfile]: yes, you are right. Fixing.

**Comment by mrunalp** [comment]
@crosbymichael @tianon PTAL. I guess we will have to merge this before drone could be updated.

**Comment by tianon** [comment]
LGTM

**Comment by rjnagal** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**